### ISSUE TITLE
VisionExcelConcerter get methods only work of English files

### ISSUE THREAD

**Issue by jaapschoutenalliander** [alliander-opensource/power-grid-model-io#143]
In the get methods of the VisionExcelConverters, english hardcoded names are used:
https://github.com/alliander-opensource/power-grid-model-io/blob/dc571487d43e6c49b8a9f31b3443b3f80cda1172/src/power_grid_model_io/converters/vision_excel_converter.py#L39
-> Number is only correct for English files and in Dutch files would be Nummer.

I would like this to be able to be retrieved from the mapping file. Like
```
        identifier = dict(self._mapping.instances(table))['line']['id']['auto_id']['key']
        return self.get_id(table=table, key={identifier: number})
```
However, I was struggling with how to generalize this for all branch types. Now 'line' is fixed, but for Links or Transformers this should be changed.

**Comment by pkuppens** [comment]
@jaapschoutenalliander , @TonyXiang8787 : I have applied for: https://inhuurdeskalliander.nl/job-requisition/scientific-software-engineer/ , and one of the tasks would be to work on the power-grid-model-io repository.

To check if I would love to do that, I checked the repo, so a few hours later, I think I have a solution for this bug.

However, I need some pointers (e.g. to get the access rights) to push my feature branch back to the repository.
